fuse_publishers/src/serialized_publisher.cpp

```diff
@@ -67,7 +67,7 @@ void SerializedPublisher::onInit()
   bool latch = false;
   private_node_handle_.getParam("latch", latch);
 
-  ros::Duration graph_throttle_period{ 0.0 };
+  rclcpp::Duration graph_throttle_period{ RCUTILS_S_TO_NS(0.0) };
```

Comment:
same comment about `RCUTILS_S_TO_NS` not being necessary here.

Reply:
https://github.com/locusrobotics/fuse/pull/283/commits/9314b85ef282a2279c816c771a8d90f591f292d7